dist/dt.html

```diff
@@ -3,45 +3,51 @@
 <head>
     <meta charset="utf-8">
     <title>DigiTru.st Identity</title>
-    <script src='./digitrust.min.js'></script>
+    <!-- DT Frame Code -->
+    <script src='./digitrust_iframe.min.js'></script>
     <script>
-    // Place initializer inside a closure
+        // Place initializer inside a closure
 
         (function () {
 
-            // ATTACH GLOBAL ERROR HANDLER
-            try {
-                var errGlobal = function(msg, url, line, col, error) {
-                    var errMsg = "Digitrust frame error: " + msg;
-                    try {
-                        if (console && console.error) {
-                            console.error(errMsg);
-                        }
-                        else {
-                            console.log(errMsg);
-                        }
-
-                        var eobj = {
-                            version: 1,
-                            type: 'DigiTrust.iframe.error',
-                            value: {}
-                        }
-                        window.parent.postMessage(eobj, document.referrer);
+            // Global error handler
+            var errGlobal = function (msg, url, line, col, error) {
+                var errMsg = "Digitrust frame error: " + msg;
+                try {
+                    if (console && console.error) {
+                        console.error(errMsg);

```

Comment:
change to Logger.error() ? need to avoid direct console logging in production.

Reply:
Yes. I need to wire in a separate initialization of the logger.

Reply:
I did add a logger object, but upon review I feel these statements should stay in place. This code lies within a global exception handler within the iFrame html. This will only trigger on unhandled exceptions in the iFrame, which should never happen or indicates a new error due to changes in browser behavior.

It is my position that we want this piece of code to fail loudly and quickly so that the underlying cause can be identified and addressed. If this particular error is hidden inside the logger we may have DigiTrust library failures go for an extended period of time unnoticed. I'd rather make publishers mad and fix the issue within a day or two than have everyone happy, but the library not work for weeks at a time.

Reply:
👍 I'm willing to try